src/App.tsx

```diff
@@ -17,18 +46,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={status}
+                query={query}
+                onQueryChanged={value => setQuery(value)}
+                onStatusChanged={value => setStatus(value)}
```

Comment:
```suggestion
                onQueryChanged={setQuery}
                onStatusChanged={setStatus}
```
redundant function wrappers

Reply:
fix everywhere